common/deployment/src/main/java/io/quarkiverse/amazon/common/deployment/DevServicesLocalStackProcessor.java

```diff
@@ -326,6 +322,7 @@ public String getSecretKey() {
                         });
 
                         ConfigureUtil.configureSharedNetwork(container, devServiceName);
```

Comment:
This utility method generates a hostname and adds it as an alias. It returns the generated host name.

Other Quarkus extensions use this generated hostname in their endpoint configuration when using a shared network, so that the Quarkus application is able to connect to the service over the shared network (vs by binding to host ports). See `MariaDBDevServicesProcessor` https://github.com/quarkusio/quarkus/blob/95c58a856c6461304d3be100aa056d5756a27a1d/extensions/devservices/mariadb/src/main/java/io/quarkus/devservices/mariadb/deployment/MariaDBDevServicesProcessor.java#L113-L130.

Can we take the same approach with this extension? Doing so allows for a Quarkus application running in a container to connect to localstack as part of a `QuarkusIntegrationTest` suite.